servlet.py

```diff
@@ -212,14 +258,25 @@ def get(self):
                 if self.get_arguments('include_deprecated_codes'):
                     responseData.append({'sourceLanguage': toAlpha2Code(l1), 'targetLanguage': toAlpha2Code(l2)})
             self.sendResponse({'responseData': responseData, 'responseDetails': None, 'responseStatus': 200})
+        elif query == 'chains':
+            src = self.get_argument('src', default=None)
```

Comment:
Other endpoints use `l1` etc., I thought `src` was more descriptive and readable. Putti already advised me to change other `l1`s in my code to better names.